api/settings.py

```diff
@@ -0,0 +1,128 @@
+"""
+Django settings for api project.
+
+Generated by 'django-admin startproject' using Django 3.1.7.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/3.1/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/3.1/ref/settings/
+"""
+
+from pathlib import Path
+
+# Build paths inside the project like this: BASE_DIR / 'subdir'.
+BASE_DIR = Path(__file__).resolve().parent.parent
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/3.1/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = ')ou55yz1axl%pv$s2b50gmmuqc0!$trok!5wv4_=x*bf@w#^2v'
+
+# SECURITY WARNING: don't run with debug turned on in production!
+DEBUG = True
+
+ALLOWED_HOSTS = []
+
+
+# Application definition
+
+INSTALLED_APPS = [
+    'django.contrib.admin',
+    'django.contrib.auth',
+    'django.contrib.contenttypes',
+    'django.contrib.sessions',
+    'django.contrib.messages',
+    'django.contrib.staticfiles',
+
+    'rest_framework',
+    'django_filters',
+
+    'calls',
+]
+
+MIDDLEWARE = [
+    'django.middleware.security.SecurityMiddleware',
+    'django.contrib.sessions.middleware.SessionMiddleware',
+    'django.middleware.common.CommonMiddleware',
+    'django.middleware.csrf.CsrfViewMiddleware',
+    'django.contrib.auth.middleware.AuthenticationMiddleware',
+    'django.contrib.messages.middleware.MessageMiddleware',
+    'django.middleware.clickjacking.XFrameOptionsMiddleware',
+]
+
+ROOT_URLCONF = 'api.urls'
+
+TEMPLATES = [
+    {
+        'BACKEND': 'django.template.backends.django.DjangoTemplates',
+        'DIRS': [],
+        'APP_DIRS': True,
+        'OPTIONS': {
+            'context_processors': [
+                'django.template.context_processors.debug',
+                'django.template.context_processors.request',
+                'django.contrib.auth.context_processors.auth',
+                'django.contrib.messages.context_processors.messages',
+            ],
+        },
+    },
+]
+
+WSGI_APPLICATION = 'api.wsgi.application'
+
+REST_FRAMEWORK = {
+    'DEFAULT_FILTER_BACKENDS': ('django_filters.rest_framework.DjangoFilterBackend',)
+}
+
+# Database
+# https://docs.djangoproject.com/en/3.1/ref/settings/#databases
+
+DATABASES = {
```

Comment:
Nas definições das variáveis de ambiente de banco de dados, sugiro o uso da https://github.com/jacobian/dj-database-url pois a mesma facilita pegar também das variáveis de ambiente e facilita ter um padrão que desejar na sua máquina e não o mesmo em produção.

Reply:
Eu instalei ele pelo poetry add, escrevi em settings.py o import dj_database_url (linha 15) e setei ele na linha 86. Contudo, o pylint acusa erro em ambas linhas. 
Linha 15: Unable to import 'dj_database_url' e 
Linha 86: Using variable 'DATABASES' before assignmentpylint(used-before-assignment)
Dei uma pesquisada mas não consegui ainda resolução

Reply:
sobre o import pode ser alguma treta do virtualenv, entao tenta os lances abaixoi:

```shell
# remover a pasta do ambiente virtual do python para esse projeto
rm -rf "$(dirname "$(dirname "$(poetry run which python)")")"
``` 

```shell
#  rodar o poetry install novamente para ele atualizar e instalar as dependencias corretamente
poetry installl
```

Se isso não funcionar precisamos ver outro caminho.